order/buyer/issue.md

```diff
@@ -0,0 +1,133 @@
+{% endapi-method-body-parameters %}
+{% endapi-method-request %}
+{% api-method-response %}
+{% api-method-response-example httpCode=200 %}
+{% api-method-response-example-description %} Successfully queued order (no body will be returned) {% endapi-method-response-example-description %}
+{% endapi-method-response %}
+{% endapi-method-spec %}
+{% endapi-method %}
+
+ [Send order OpenAPI Specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-integration/specs.yaml#/order-integration/sendOrderByBuyerRoute)
+
+### Order body JSON objects
+
+#### Order
+
+- `companyId`: your Tradecloud company identifier. You can find your company id in the URL when selecting "My company" in the portal dropdown menu. For example in `https://portal.accp.tradecloud1.com/company/06893bba-e131-4268-87c9-7fae64e16ee9` the last part `06893bba-e131-4268-87c9-7fae64e16ee9` is the company id.
+- `supplierAccountNumber`: the supplier account number as known in your ERP system
+- `purchaseOrderNumber`: the purchase order number as known in your ERP system
+- `destination`: the delivery destination of this order as known in your ERP system
+- `contact`: the employee responsible for this order. You can either send his/her email or userName as known in your ERP system
+  
+{% hint style="danger" %}
+`supplierAccountNumber`, `purchaseOrderNumber`, `destination.code`, `contact.email` and `contact.userName` should be unique within your company and never change. Never renumber or re-use numbers or code's.
+{% endhint %}
+
+{% hint style="danger" %}
+The `supplierAccountNumber` should be set on forehand in the Tradecloud connection with your supplier. You can set the account code when inviting a new connection or at any time in the connection overview in the portal.
+{% endhint %}
+
+#### Secondary order fields
+
+- `description`: a free format additional description of this order
+- `terms`: the order terms as agreed with your supplier
+- `indicators`: see [Indicators](indicators.md)
+- `properties`: are key-value based custom fields. You can user as many as needed, but too many will clutter the portal. Use `\n` for a new line in the value.
+- `notes`: are simple custom fields. You can user as many as needed, but too many will clutter the portal. Use `\n` for a new line.
+- `documents`: contain meta data and link of attached documents. See [Attach a document to an order](attach-document.md)
+
+#### Lines
+
+- `lines`: a purchase order contains one or multiple lines
+- `line`: a purchase order line contains at least the position, item and delivery schedule
+- `position`: the line position within the purchase order
+
+{% hint style="danger" %}
+`lines.position` should be unique within the order and never change.
+Never renumber or re-use `position` numbers.
+{% endhint %}
+
+#### Item
+
+- `lines.item`: the item (or article, goods) to be delivered
+- `lines.item.number`: the item code or number as known in your ERP
```

Comment:
Generally speaking; don't we want these field explanation in our Swagger specs, and reference to those? This will reduce maintenance and keep this page short.

Reply:
The model in the Swagger specs is hidden and ugly rendered. I want to present a complete 'story' here to prevent support requests and point to the model again and again.